### PR TITLE
Fix/242 fix gradle warnings

### PR DESCRIPTION
**Pull request by marchuk0**

### CODE REVIEW

**Review by TheBestTvarynka — CHANGES_REQUESTED** [review]
I run this branch on my machine and I get the following error: 
```
Parameter 0 of constructor in com.feed_grabber.core.fileStorage.AmazonService required a bean of type 
'com.amazonaws.services.s3.AmazonS3Client' that could not be found.
```

**Review by TheBestTvarynka — APPROVED** [review]
All warnings fixed. :+1:

File: backend/core/src/main/java/com/feed_grabber/core/config/S3Config.java [hunk 1]
```diff
@@ -15,9 +18,12 @@
...
     @Bean
-    public AmazonS3Client generateS3Client() {
+    public AmazonS3 generateS3Client() {
         AWSCredentials credentials = new BasicAWSCredentials(accessKey,accessSecret);
-        AmazonS3Client client = new AmazonS3Client(credentials);
-        return client;
+        return AmazonS3ClientBuilder
+                .standard()
+                .withCredentials(new AWSStaticCredentialsProvider(credentials))
+                .withRegion(Regions.EU_CENTRAL_1)
```
Comment (MaksymSlobodianyk) [inline backend/core/src/main/java/com/feed_grabber/core/config/S3Config.java:26]: Change region to bucket region

**Review by MaksymSlobodianyk — APPROVED** [review]

File: backend/core/src/main/java/com/feed_grabber/core/config/S3Config.java (hunk 1 above)
Comment (MaksymSlobodianyk) [reply backend/core/src/main/java/com/feed_grabber/core/config/S3Config.java:26]: Now upload works!

**Pull request merged**